docs/learn/computation/execution-model.md

```diff
@@ -12,12 +12,40 @@ When a transaction is executed successfully, it is included in the current block
 
 When block generation is complete, the block is propagated to all the other CNs. The other CNs all verify the propagated block and reach consensus on the verification results by using the BFT consensus algorithm. When the verification process completes successfully by the majority of CNs, the block is stored in the blockchain. Because the BFT consensus algorithm satisfies the immediate finality property, the block is final and will never be removed. After a block is finalized, the execution of all the transactions in that block are irreversibly guaranteed, and their execution results can be returned to the sender if requested.
 
+## Enhanced Randomness in Block Proposer Selection and Reward Distribution <a id="enhanced-randomness-in-block-proposer-selection-and-reward-distribution"></a>
+
+Kaia has implemented a new mechanism to introduce verifiable on-chain randomness in the block proposer selection and reward distribution processes. This mechanism involves two new fields in the block header: `randomReveal` and `mixHash`.
+
+In this system, validators generate and commit to random values. The `randomReveal` field in a block contains the revealed random value from a previous commitment. The `mixHash` is then computed using this revealed random value along with other block data, creating a source of randomness for the network.
```

Comment:
The `randomReveal` is calculated from the signature of block proposer upon current mining block number, not from a previous committment. Please check the this [KIP](https://kips.klaytn.foundation/KIPs/kip-114)

Reply:
@hyeonLewis Thank you for the feedback. Please check the revised definition of 'randomReveal' in my [first revision](https://github.com/kaiachain/kaia-docs/pull/69/commits/617a4d15e73e432a05605b177a22bfe11af4116f) commit if it's correct.